src/Minigames/crew.js

```diff
@@ -131,13 +136,14 @@ Commands.crew = new Command("Find the imposter!\n\n" + Amongus.help + "\npossibl
     }
     Amogus.turns = Math.max(Amogus.turns - 1, 0)
     if (Amogus.turns == 0) {
-        message.channel.send("The impostor killed you!\nThe impostor was " + Amogus.getSussier() + ".\nGame over.")
+        message.channel.send("The impostor killed you!\nThe impostor was " + Amogus.getSussiest() + ".\nGame over.")
         let EconomySystem = Economy.getEconomySystem(message.author)
         EconomySystem.steal(30 + (10 * aturn), message)
-        Amogus.reset()
+        delete Amogus
```

Comment:
`delete` does not delete a variable.

Reply:
wait wtf, why delete even if?
it should reset the values to default, not straight up delete it...?

Reply:
fair 
I wasn't sure if should be doing that, lemme rollback